pyproject.toml

```
@@ -85,3 +85,8 @@ qulacs = {version = "0.1.10.1", markers = "platform_machine == 'x86_64'" }
 
 # The following packages are only installed on MacOS for compatibility
 tensorflow-macos = { version = "2.8.0", markers = "sys_platform == 'darwin' and platform_machine == 'arm64'" }
+
```

```suggestion

[tool.poetry.group.metadata-validation]
optional = true

```

So the metadata validation group isn't installed by default